scripts/translate

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -eu
+
+apps='core frontend'
+
+for lang in "$@"; do
```

Comment:
If no parameter is given, the script will just exit silently. Maybe it would be nice adding a small check if 'lang' is empty

Reply:
nice catch. I added this check, and removed the loop; it will only take one language at a time.

Reply:
ack